feat: add option `skipArtifactsUpdate` (#35993)
renovatebot/renovate: docs/usage/configuration-options.md

But if you set `separateMultipleMinor=true` then you get separate PRs for each minor stream, like `python@3.9.x`, `python@v3.10.x` and `python@v3.11.x`, etc.

## skipArtifactUpdating

> Use this option in cases where automatic artifact updating can be problematic, or when you would like to customize how artifacts (such as lock files) are updated. When this option is set, you should update artifacts (such as lock files), via `postUpgradeTasks` or another method such as a CI workflow.

Reviewer: Add a note that all upgrades in a branch (aka grouping) needs this have to be true to really skip.

But if you set `separateMultipleMinor=true` then you get separate PRs for each minor stream, like `python@3.9.x`, `python@v3.10.x` and `python@v3.11.x`, etc.

## skipArtifactUpdating

> Use this option in cases where automatic artifact updating by Renovate fails or is incorrect, or you have custom needs.

Reviewer 2: ```suggestion
Use this option when automatic artifact updating fails, is incorrect, or not needed.  
This is useful if you have custom workflows handling artifact updates.

```

Author: @rarkins Do you want this change, or stick with what you previously suggested?
https://github.com/renovatebot/renovate/pull/35993/files/98a1e917c59b34176c3795abcdb834d9a95c6b6d#r2105764636

> When this option is set, Renovate won't attempt to update artifacts such as lock files, so you will need to update them yourself, either manually or through secondary automation such as CI workflows.

Reviewer 2: ```suggestion
When this option is set, Renovate won't attempt to update artifacts such as lock files.
 You will need to update them yourself, either manually or via secondary automation such as CI workflows.
```

But if you set `separateMultipleMinor=true` then you get separate PRs for each minor stream, like `python@3.9.x`, `python@v3.10.x` and `python@v3.11.x`, etc.

## skipArtifactsUpdate

> 

Reviewer: add a note that this was formerly known as `updateLockFiles`
